### DISCUSSION TITLE
Importing .shp files

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#212] [Ideas]
Hi Magne, 
a last question for today.
Do you consider enabling the direct import of .shp files?
It would be interesting as I have made a huge data entry in this format.
Best Regards
Federico

**Comment by kjellmf** [comment]
shp files are not natively supported by OpenLayers, but it should be possible to import .shp files using another library. I've added an issue to keep track of this #213

**Reply by kjellmf** [reply]
Reading shapefiles was harder that I thought. It may be easier to just use an online conversion tool.
